spec/system/search_game_rooms_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+RSpec.describe 'SearchGameRooms', type: :system do
+  let(:user) { build(:user, points: 100) }
+  let(:track) { create(:track) }
+  let(:room) { create(:room) }
+
+  before do
+    driven_by(:rack_test)
+    user.skip_confirmation!
+    user.save!
+    sign_in user
+    room
+    track
+    room.tracks << track
```

Comment:
### Note 2
Since we already associated the tracks to the room we can omit these 😄 
```suggestion
```